packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -20,12 +20,12 @@ export type PopperPlacementType =
 export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
   transition?: boolean;
   anchorEl?: null | HTMLElement | ReferenceObject | ((element: HTMLElement) => HTMLElement);
-  children: (
+  children: React.ReactElement<any> | ((
```

Comment:
It can be a node, I believe we should be using `React.ReactNode`.

Reply:
done